kubel.el

```diff
@@ -328,8 +328,8 @@ READONLY If true buffer will be in readonly mode(view-mode)."
   (when (get-buffer buffer-name)
     (kill-buffer buffer-name))
   (if async
-      (apply #'start-process buffer-name buffer-name "kubectl" (append (kubel--get-context-namespace) args))
-    (apply #'call-process "kubectl" nil buffer-name nil (append (kubel--get-context-namespace) args)))
+      (apply #'start-file-process buffer-name buffer-name "kubectl" (append (kubel--get-context-namespace) args))
```

Comment:
use 'start-file-process' and 'process-file' to run a process in a remote directory